bears/general/FilenameBear.py

```diff
@@ -38,6 +46,21 @@ def run(self, filename, file,
         """
         head, tail = os.path.split(filename)
         filename_without_extension, extension = os.path.splitext(tail)
+
+        if file_naming_convention is None:
+            self.err('Please specify a file naming convention explicitly. '
```

Comment:
IMO, this is more of a warning, since we are falling back to something default. It should be an error if the bear can't run.